internal/mocks/mock_cache.go

```diff
@@ -20,7 +20,6 @@ import (
 type MockInMemoryCache[T any] struct {
 	ctrl     *gomock.Controller
 	recorder *MockInMemoryCacheMockRecorder[T]
-	isgomock struct{}
```

Comment:
🤔

Reply:
@elbuo8 this was generated after the last rebase

Reply:
@elbuo8 here's the cause: https://github.com/uber-go/mock/pull/204

I was the first to generate mocks